### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by leokraemer**
add readme to core that outlines how to use it

### CODE REVIEW

File: lib/WikidataIdListGenerator [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 0e5c0e282190c4e02c83196262525e30082c78ed
+Subproject commit 6a4779f945d7c22bbde2bd94e160742ab44e679b
```
Comment (physikerwelt) [inline lib/WikidataIdListGenerator]: What was updated here? To which project does that refere?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,13 @@ We are using the [NTCIR-11 Wikipedia](http://ntcir11-wmc.nii.ac.jp/index.php/NTC
 (specifically the augmentedWikiDump.xml from [this host](http://demo.formulasearchengine.com/images/)) for as additional
 training dataset.
 
+### Checkout
+  
+  * if test fail on windows make sure to checkout the files with unix-style line endings. This is most easily done by
```
Comment (physikerwelt) [inline README.md]: tests?

File: lib/WikidataIdListGenerator (hunk 1 above)
Comment (physikerwelt) [inline lib/WikidataIdListGenerator]: ok this is the upstream head versioin

File: README.md (hunk 2 above)
Comment (leokraemer) [inline README.md]: the performance test fail when the test/resources have the wrong line endings

File: lib/WikidataIdListGenerator (hunk 1 above)
Comment (leokraemer) [inline lib/WikidataIdListGenerator]: should i revert to the old version?

File: README.md (hunk 2 above)
Comment (leokraemer) [inline README.md]: for example basex/src/test/java/clienttest contains final String expectedOutput = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
			"<results xmlns=\"http://ntcir-math.nii.ac.jp/\">\n" +
			"  <run runtag=\"\" run_type=\"\">\n" +
			"    <result for=\"NTCIR11-Math-\">\n" +
			"      <hit id=\"8#math.8.13\" xref=\"\" score=\"10\" rank=\"1\"/>\n" +
			"      <hit id=\"8#math.8.22\" xref=\"\" score=\"10\" rank=\"2\"/>\n" +
			"      <hit id=\"8#math.8.23\" xref=\"\" score=\"10\" rank=\"3\"/>\n" +
			"      <hit id=\"8#math.8.25\" xref=\"\" score=\"10\" rank=\"4\"/>\n" +
			"      <hit id=\"8#math.8.25\" xref=\"\" score=\"10\" rank=\"5\"/>\n" +
			"      <hit id=\"8#math.8.30\" xref=\"\" score=\"10\" rank=\"6\"/>\n" +
			"      <hit id=\"8#math.8.32\" xref=\"\" score=\"10\" rank=\"7\"/>\n" +
			"    </result>\n"+
			"  </run>\n" +
			"</results>";

This does not work with windows style line endings

File: lib/WikidataIdListGenerator (hunk 1 above)
Comment (physikerwelt) [inline lib/WikidataIdListGenerator]: no it's fine

**Pull request merged**